github/Repository.py

```diff
@@ -1867,6 +1867,22 @@ def get_forks(self):
             None
         )
 
+    def create_fork(self, organization=github.GithubObject.NotSet):
+        """
+        :calls: `POST /repos/:owner/:repo/forks <http://developer.github.com/v3/repos/forks>`_
+        :param organization: string or "none" or "*"
+        :rtype: :class:`github.Repository.Repository`
+        """
+        post_parameters = {}
```

Comment:
We also assert arguments are correct, sorry I failed to pick that up before.

Reply:
thanks will do

Reply:
added now